stock_lot_removal_date/readme/CONFIGURE.md

```diff
@@ -0,0 +1,3 @@
+Set timezone of the superuser (OdooBot) according to the main location
+of the location, which will affect how Removal Date is set in
+lot/serial.
```

Comment:
Please update this according to the latest design. We may mention that OCA module partner_tz can be used to display the time zone field in the partner form.

Reply:
I updated!